beacon-chain/powchain/log_processing.go

```diff
@@ -184,6 +184,13 @@ func (s *Service) ProcessDepositLog(ctx context.Context, depositLog gethTypes.Lo
 			"merkleTreeIndex": index,
 		}).Debug("Deposit registered from deposit contract")
 		validDepositsCount.Inc()
+		// Notify users what is going on, from time to time.
+		if !s.chainStartData.Chainstarted {
```

Comment:
How is this different than the following log?
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/powchain/log_processing.go#L185

Do you want to remove that? They seem similar

Reply:
The one you linked provides per-deposit information, including the public key of the deposit.  This one provides more of a progress meter, showing only one report every 512 deposits and no detailed information.  I think they both have a place.

Reply:
Got it. 👍